src/inc/stackstring.h

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+
+#ifndef __STACKSTRING_H_
```

Comment:
Including SString derived types in the Wrapper APIs was causing a lot of trouble because of the includes we have in sstring.h, that is the reason I opted to use this stripped down version. If there are ways in which we could share the code between PAL and VM, I would like to know how ?